meta/parse.pl

```diff
@@ -359,6 +359,8 @@ sub ProcessTagDefault
 
     return $val if $val eq "\"\"";
 
+    return $val if $val =~ /^0\,0$/;
```

Comment:
not need for \\, since comma is not regex special

Reply:
Corrected